service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java

```diff
@@ -53,8 +64,15 @@ public void shouldMakeCorrectCallForCtsc(Boolean toggleState) {
         givenToggle(toggleState);
 
         assertThat(featureToggleService.isCtscEnabled("test name")).isEqualTo(toggleState);
+        featureToggleService.isCtscReportEnabled();
 
-        verify(ldClient).boolVariation(eq("CTSC"), any(LDUser.class), eq(false));
+        verify(ldClient, atLeastOnce()).boolVariation(eq("CTSC"), ldUser.capture(), eq(false));
+
+        LDUser expectedLdUser = getLdBuilder()
+            .custom("localAuthorityName", "")
+            .build();
+
+        assertThat(ldUser.getAllValues().get(0).getCustomAttributes()).isEqualTo(expectedLdUser.getCustomAttributes());
```

Comment:
getAllvalues can be replaces with getLastValue (or getValue) which makes the assertion nicer. .

Reply:
note that this will change the assertion as `getValue()` calls get last value and as the verify captures 2 calls the last value captured will be from `featureToggleService.isCtscReportEnabled()` instead of `featureToggleService.isCtscEnabled("test name")`